### ISSUE TITLE
pdv_functions

### ISSUE THREAD

**Issue by rootsmusic** [nonpartisan-redistricting-datahub/pdv-resources#4]
@peter-redistricting-data-hub Where can I find pdv_functions?  Thanks.

**Comment by peter-redistricting-data-hub** [comment]
They were hidden away inside a Jupyter notebook, but I just uploaded them as a separate Python file

**Comment by rootsmusic** [comment]
Ok, I thought it'd be published at https://pypi.org/

**Comment by peter-redistricting-data-hub** [comment]
I haven't launched a package before, have you?

**Comment by rootsmusic** [comment]
No, but see [Python Packaging User Guide](https://packaging.python.org/en/latest/tutorials/packaging-projects/).  I'm looking up how to import a .py file.

**Comment by rootsmusic** [comment]
@peter-redistricting-data-hub please cleanup functions with the same name in https://github.com/nonpartisan-redistricting-datahub/pdv-resources/blob/main/Notebook_Report_Guidelines.ipynb as those in pdv_functions.

**Comment by rootsmusic** [comment]
@peter-redistricting-data-hub Please cleanup functions with the same name in Notebook_Report_Guidelines.ipynb as those in pdv_functions.py, unless pdv_functions.py won't be in this repo when it's published elsewhere.

**Comment by peter-redistricting-data-hub** [comment]
I'll try to get that to that this week, thanks for the heads up
